pom.xml

```diff
@@ -51,7 +51,7 @@
         <skipUnitTests>false</skipUnitTests>
 
         <!-- The following properties configure the embedded MongoDB instance used for tests -->
-        <embeddedMongoDbVersion>2.6.5</embeddedMongoDbVersion>
+        <embeddedMongoDbVersion>3.0.2</embeddedMongoDbVersion>
```

Comment:
Should we have another profile for 2.6 so we can run tests against it on CI? It's a more general question I guess, should we only test against latest major versions of backends or also previous ones.